### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -1867,7 +1867,7 @@ msgstr "Невірно процитований вміст у файлі \"%s\":
 #: bisect.c:702
 #, c-format
 msgid "We cannot bisect more!\n"
-msgstr "Неможливо бісектувати більше!\n"
+msgstr "Подальше бісектування неможливо!\n"
```
Comment (kasya) [inline po/uk.po:1870]: 🔥

File: po/uk.po [hunk 2]
```diff
@@ -3682,71 +3686,71 @@ msgstr "налаштувати конфігурацію відстежувани
...
 
 #: builtin/branch.c:701
 msgid "copy a branch and its reflog"
-msgstr ""
+msgstr "скопіювати гілку та її лог посилань"
 
 #: builtin/branch.c:702
 msgid "copy a branch, even if target exists"
-msgstr ""
+msgstr "скопіювати гілку гілку, навіть якщо призначення існує"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "скопіювати гілку, навіть якщо призначення існує"
```

File: po/uk.po [hunk 3]
```diff
@@ -3758,41 +3762,41 @@ msgstr "показати назву поточної гілки"
...
 
 #: builtin/branch.c:714
 msgid "print only branches of the object"
-msgstr ""
+msgstr "виводити тільки гілки об’єкта"
 
 #: builtin/branch.c:715 builtin/for-each-ref.c:58 builtin/tag.c:493
 msgid "sorting and filtering are case insensitive"
-msgstr ""
+msgstr "сортувати та фільтрувати незалежно від регістру"
```
Comment (kasya) [inline po/uk.po:3799]:
```suggestion
msgstr "сортування та фільтрування не залежить від регістру"
```

File: po/uk.po [hunk 4]
```diff
@@ -3816,48 +3820,52 @@ msgid ""
...
 
 #: builtin/branch.c:860
 msgid "too many arguments for a rename operation"
-msgstr ""
+msgstr "забагато аргументів для операції перейменування"
 
 #: builtin/branch.c:871
 msgid "too many arguments to set new upstream"
-msgstr ""
+msgstr "забагато аргументів для встановлення нового першоджерельне сховище"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "забагато аргументів для встановлення нового першоджерельного сховища"
```

**Review by kasya — COMMENTED** [review]
LGTM!👍🏼

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po:1870]: 🎯

File: po/uk.po (hunk 3 above)
Comment (arkid15r) [reply po/uk.po:3799]: The first option seems better to me

File: po/uk.po [hunk 5]
```diff
@@ -24405,8 +24463,8 @@ msgstr "збережений pre-cherry-pick HEAD файл \"%s\" пошкодж
 #: sequencer.c:3306
 msgid "You seem to have moved HEAD. Not rewinding, check your HEAD!"
 msgstr ""
-"Здається, ви пересунули HEAD. Перемотування не виконується, перевірте свій "
-"HEAD!"
+"Здається, ви пересунули HEAD. Перемотування  вперед не виконується, "
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Здається, ви пересунули HEAD. Перемотування назад не виконується, "
```

**Pull request merged**